data/presets/presets/attraction/log_flume.json

```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-tower",
```

Comment:
We should probably pick a better icon for a log flume.. Maybe `maki-ferry` for now until we come up with a better one.

Reply:
Thank you for pointing that out, I just pushed a commit changing the icon. 👍